pkg/runtime/cache/cache.go

```diff
@@ -43,6 +43,12 @@ const (
 	// caching system not to set up resyncs with an authoritative state source
 	// (i.e. a Kubernetes API server) on a periodic basis.
 	informerResyncPeriod = 0 * time.Second
+
+	// The prefix for owner account ID in the v2 CARM map.
+	OwnerAccountIDPrefix = "owner-account-id."
+
+	// The prefix for owner team ID in the v2 CARM map.
+	TeamIDPrefix = "team-id."
```

Comment:
QQ: are we assuming that users will populate the configmap with something like `team-id.abc` and `owner-account-id.012345678912` ?  trying to understand the liaison with https://github.com/aws-controllers-k8s/community/issues/2031

Reply:
IMO the format should exactly be `s3.abc` or `s3.012345678912` - adding `team-id` or `owner-account-id` makes things a little unusual. However mind proposing this new format in the Github issue? maybe i'm wrong and ACK users would like to have it this way :)